components/automate-cluster-ctl/habitat/plan.sh

```diff
@@ -18,11 +18,11 @@ pkg_deps=(
   core/gzip
   core/jq-static
   core/openssh
-  core/openssl
+  core/openssl/1.0.2zb/20220311111046
   core/make
   core/curl
-  core/rsync
-  core/terraform/0.14.8/20210826165930
+  core/rsync/3.2.3/20220817235706
```

Comment:
is  the rsync pkg got updated in builder and with the new pkg with build is failing then pining the pkg is fine other wise we need to unpin the pkg and re-test the senariao

Reply:
The pkg was used for testing purposes. I have tested the same with the default core/rsync package.